src/value_generators/src/DocumentGenerator.cpp

```diff
@@ -153,6 +153,9 @@ using Parser = std::function<O(const Node&, GeneratorArgs)>;
 
 // Pre-declaring all at once
 // Documentation is at the implementations-site.
+const static int64_t parseStringToMillis(const std::string& datetime);
+const static boost::posix_time::ptime epoch(boost::gregorian::date(1970, 1, 1));
```

Comment:
plz use `=` or curlies to make it clear these are static members not forward references.